### PR TITLE
Mobile optimized homepage

### PR DESCRIPTION
**Pull request by gvn**
Related issue: #1299

### CODE REVIEW

**Comment by gvn** [comment]
Screenshots for design review (since we don't have dummy data for Wagtail on staged PRs yet):

localhost_8000_cms_pages_3_view_draft_ 1
localhost_8000_cms_pages_3_view_draft_

**Comment by kristinashu** [comment]
Ooh you did a lot! Looks great!

**Comment by gvn** [comment]
Alan or Mavis just remove the other person if you take this review...not sure who has time.

File: source/sass/utilities.scss [hunk 1]
```diff
@@ -57,3 +57,35 @@
...
+
+.full-bleed {
+  @include full-bleed;
+}
+
+@include full-bleed-bp("xs");
+@include full-bleed-bp("sm");
+@include full-bleed-bp("md");
+@include full-bleed-bp("lg");
+@include full-bleed-bp("xl");
```
Comment (mmmavis) [inline source/sass/utilities.scss]: oh neat

**Pull request merged**